src/display.hpp

```diff
@@ -767,6 +767,7 @@ class display : public gui2::top_level_drawable
 	mutable events::generic_event scroll_event_;
 
 	boost::circular_buffer<unsigned> frametimes_; // in milliseconds
+	int current_frame_sample = 0;
```

Comment:
The variable name should end in underscore like the other member variables.

Reply:
https://github.com/wesnoth/wesnoth/pull/6965